docs/api-overview.mdx

```diff
@@ -197,7 +203,16 @@ Let's break this down to understand what is happening here:
 }
 ```
 
-> _¹The `FullNameCredential` also has a `MiddleNameCredential`, but it's `optional`._
+
+- `PhoneCredential` is a "[single attribute atomic credential](/schema#schema-definitions)" and therefore does not have any sub-credentials. Because we set a `allowUserInput` attribute of `true`, the user will be given an opportunity to enter a new value if they choose."
+
+```json title="PhoneCredential"
+{
```

Comment:
The `mandatory` attribute is missing

Reply:
There is a task to make a combined check to allow user to input instead of just `allowUserInput`: https://trello.com/c/D4uhDaWt/5971-allow-user-to-input-only-if-allowuserinput-is-true-and-mandatory-is-ifavailable-changes-to-new-wallet-ui

Reply:
@iagormoraes so you are saying `mandatory` is not needed if `allowUserInput` is `true`? If that is the case, could you please explain that in the docs? Either way, I would normally lean towards being more explicit when it comes to inputs 
like this. The implicit nature is where confusion happens. 

_Apologies this PR has sat so long open!_